typescript-arrays/main.ts

```diff
@@ -0,0 +1,27 @@
+const colors: string[] = ['red', 'white', 'blue'];
+const firstColor: string = colors[0];
+const secondColor: string = colors[1];
+const thirdColor: string = colors[2];
+console.log('value of colors[0]:', firstColor);
+console.log('value of colors[1]:', secondColor);
+console.log('value of colors[2]:', thirdColor);
+
+const AmericaColors: string = `America is ${firstColor} ${secondColor} ${thirdColor}`;
+console.log(AmericaColors);
+
+colors[2] = 'green';
+const MexicoColors: string = `Mexico is ${firstColor} ${secondColor} ${thirdColor}`;
```

Comment:
You did this correctly on line 12 where you updated colors[2] to be green, but if you look at your console it shows "Mexico is red white blue" and Mexico should be red white green.